cclient/serialization/tests/test_json.c

```diff
@@ -690,15 +538,16 @@ void test_serialize_were_addresses_spent_from(void) {
   const char* json_text =
       "{"
       "\"command\":\"wereAddressesSpentFrom\","
-      "\"trytes\":[\"" WERE_ADDRESSES_SPENT_FROM_TRYTES
+      "\"addresses\":[\"" TRANSACTION_TRYTES_1 "\",\"" TRANSACTION_TRYTES_2
```

Comment:
the address is 81 trytes in IOTA protocol.

Reply:
sorry for this mistake.
I referred iota.readme.io when I wrote these code, and I just found the python version of API document is wrong. (https://iota.readme.io/reference#storetransactions)